### PR TITLE
Radarr: striptracks initial release

### PR DESCRIPTION
**Pull request by TheCaptain989**
Add mkvtoolnix and script to strip unwanted tracks
Suitable for Radarr or Sonarr

### CODE REVIEW

File: root/etc/cont-init.d/60-striptracks-init [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  if [ -f /usr/bin/apt ]; then
+    # Ubuntu
+    echo "Installing MKVToolNix using apt-get"
+    apt-get update && \
+    apt-get -y install mkvtoolnix && \
+    rm -rf /var/lib/apt/lists/*
+  elif [ -f /sbin/apk ]; then
+    # Alpine
+    echo "Installing MKVToolNix using apk"
+    apk add mkvtoolnix && \
```
Comment (aptalca) [inline root/etc/cont-init.d/60-striptracks-init:25]: should be `apk add --no-cache mkvtoolnix`

File: root/etc/cont-init.d/60-striptracks-init [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    apt-get update && \
+    apt-get -y install mkvtoolnix && \
+    rm -rf /var/lib/apt/lists/*
+  fi
+fi
+
+# Change ownership
+if [ $(stat -c '%G' /usr/local/bin/striptracks.sh) != "users" ]; then
+  echo "Changing ownership on scripts."
+  chown root:users /usr/local/bin/striptracks*.sh
```
Comment (aptalca) [inline root/etc/cont-init.d/60-striptracks-init:39]: why not `abc:abc`, which is what all the apps run as inside the container?

File: root/etc/cont-init.d/60-striptracks-init [hunk 3]
```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/with-contenv bash
```
Comment (aptalca) [inline root/etc/cont-init.d/60-striptracks-init:1]: this init script probably should be named `98-striptracks` for uniformity with all other mods unless there is a specific reason to run this earlier

File: Dockerfile [hunk 4]
```diff
@@ -1,6 +1,46 @@
...
+# Build arguments
+ARG BUILD_DATE
+ARG VCS_REF
+ARG VERSION
+ARG DOCKERHUB
+ARG BASEIMAGE
+ARG MODNAME
+
+# Build-time metadata as defined at http://label-schema.org
+LABEL org.label-schema.name="${DOCKERHUB}:${BASEIMAGE}-${MODNAME}" \
```
Comment (aptalca) [inline Dockerfile:13]: Do these labels appear in the final image? I've never tested this scenario but I would imagine they would be discarded/lost along with the buildstage

Comment (TheCaptain989) [reply Dockerfile:13]: You're right. Here's my earlier test build [here](https://microbadger.com/images/thecaptain989/striptracks-mod).  I'll remove them.

File: root/etc/cont-init.d/60-striptracks-init (hunk 1 above)
Comment (TheCaptain989) [reply root/etc/cont-init.d/60-striptracks-init]: Fixed.

File: root/etc/cont-init.d/60-striptracks-init (hunk 2 above)
Comment (TheCaptain989) [reply root/etc/cont-init.d/60-striptracks-init]: Fixed

File: root/etc/cont-init.d/60-striptracks-init (hunk 3 above)
Comment (TheCaptain989) [reply root/etc/cont-init.d/60-striptracks-init]: Fixed

File: Dockerfile (hunk 4 above)
Comment (TheCaptain989) [reply Dockerfile]: I simplified the Dockerfile and .travis.yml files, removing unused variables, accordingly.

File: README.md [hunk 5]
```diff
@@ -1,17 +1,176 @@
...
-* Ask the team to create a new branch named `<baseimagename>-<modname>`. Baseimage should be the name of the image the mod will be applied to. The new branch will be based on the `template` branch.
-* Submit PR against the branch created by the team.
+2. Configure the Docker container with all the port, volume, and environment settings from the *original container documentation* here:  
+   **[linuxserver/radarr](https://hub.docker.com/r/linuxserver/radarr "Radarr Docker container")**  
+   **[linuxserver/sonarr](https://hub.docker.com/r/linuxserver/sonarr "Sonarr Docker container")**
+   1. Add the **DOCKER_MODS** environment variable to the `docker create` command, as follows:  
+      `-e DOCKER_MODS=linuxserver/mods:radarr-striptracks`  
+
+      *Example Synology Configuration*  
+      ![striptracks](https://user-images.githubusercontent.com/11523885/95027314-1e006680-065d-11eb-9ff9-cc51648d57bd.png "Synology container settings")
```
Comment (aptalca) [inline README.md:24]: fyi, this pic references the wrong endpoint for the mod

**Comment by aptalca** [comment]
I was just about to merge it when I realized the issue with the pic.

Also while you're at it, you can drop the pics in an `.assets` folder and reference them there so we keep everything in the same repo, just like done here: https://github.com/linuxserver/docker-mods/tree/letsencrypt-f2bdiscord

File: README.md (hunk 5 above)
Comment (TheCaptain989) [reply README.md:24]: Whoops!  Good catch.  That was a reference to my test build.

**Comment by TheCaptain989** [comment]
> I was just about to merge it when I realized the issue with the pic.
> 
> Also while you're at it, you can drop the pics in an `.assets` folder and reference them there so we keep everything in the same repo, just like done here: https://github.com/linuxserver/docker-mods/tree/letsencrypt-f2bdiscord

Done.

**Comment by aptalca** [comment]
Great work, it looks excellent. Thanks for the quality pr.

**Review by aptalca — APPROVED** [review]

**Comment by TheCaptain989** [comment]
You're most welcome.  And thank you!

**Pull request merged**